libmscore/tempotext.cpp

```diff
@@ -236,7 +236,7 @@ QVariant TempoText::propertyDefault(P_ID id) const
       {
       switch(id) {
             case P_ID::TEMPO:             return 2.0;
-            case P_ID::TEMPO_FOLLOW_TEXT: return false;
+            case P_ID::TEMPO_FOLLOW_TEXT: return true;
```

Comment:
Hmm, we could uncomment the `tt.resetFollowText->setDisabled(followText);` in InspectorTempText::postInit() to solve the reset button being enabled on those tempo texts.

Where (and how) to use those writeProperty/readProperty? Or would that be in a separate change anyway?